client/components/common/StatEditor.tsx

```diff
@@ -188,6 +190,55 @@ const StatEditor: React.FC<Props> = ({ customSet, className }) => {
     },
   );
 
+  const quickEditStat = (baseKey: StatKey, amount: number) => {
+    dispatch({
+      type: 'edit',
+      stat: baseKey,
+      value: amount,
+    });
+    debouncedCheckAndMutate();
+  };
+
+  const getQuickStatButtons = (baseKey: BaseStatKey) => {
+    const getButtonValues = (level: number) => {
```

Comment:
Could we calculate these instead of hard coding them? For instance, a level 21 character would be able to put 100 points.

Also, maybe these could be a component? You're very close to making them one, why not just do it and include the Tooltip?